rpcserver.go

```diff
@@ -6129,6 +6129,7 @@ func (r *rpcServer) AddInvoice(ctx context.Context,
 		NumHops:                  globalBlindCfg.NumHops,
 		MaxNumPaths:              globalBlindCfg.MaxNumPaths,
 		NodeOmissionSet:          fn.NewSet[route.Vertex](),
+		IncomingChainedChannels:  make([]uint64, 0),
```

Comment:
can be removed, initialization with 0 makes no difference rather not init it at all.